src/test/java/core/basesyntax/FruitDaoImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.basesyntax.model.Fruit;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitDaoImplTest {
+    private FruitDaoImpl fruitDao;
+
+    @BeforeEach
+    public void setUp() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Test
+    public void testAdd() {
```

Comment:
try to be consistent in terms of method naming conventions. See common mistakes

Reply:
Sorry, really careless...